### ISSUE TITLE
Hopperhocks do not input into Botania inventories

### ISSUE THREAD

**Issue by GameGuruGreenGryphon** [Vazkii/Botania#3942]
### Mod Loader

Forge

### Minecraft Version

1.18.1

### Botania version

Botania-1.18.1-428-SNAPSHOT

### Modloader version

Forge 39.0.64

### The latest.log file

https://gist.github.com/GameGuruGreenGryphon/a804b4ea57aa5f1b8b3992f9b6350b90

### Issue description

Hopperhocks do not input any item into Botania inventories.
Hopperhocks meaning any variant of Hopperhock (petite, floating, floating petite) and any Botania inventory including any mana spreader, tiny potatoes, red-stringed blocks, open crates, crafty crates, and mana prisms.

### Steps to reproduce

1. Place any Botania inventory down.
2. Place any variant of Hopperhock with any of their settings next to said inventory, and a hopper pointing into inventory.
3. Drop target item on the ground, and a copy of said item into hopper to prove that the inventory can be input to.

### Other information

Hopperhocks output to vanilla inventories, and even other modded ones like Quark's pipes.  I did not test more mods' inventories.
This is on build number 79 on the Botania's ci.blamejared.com directory with the livingwood PR head, and the version running on the Botania/Hex playtesting server.

**Comment by williewillus** [comment]
Can you please specify which "Botania inventories" you tried? Not all of them are exposed to automation.

**Comment by williewillus** [comment]
Essentially, bug reports should have no instance of the word "any" in them. Put only what you've tried.

**Comment by Hubry** [comment]
Open crates were the original reason, looking at server chat logs

**Comment by GameGuruGreenGryphon** [comment]
The list of items I tested is in the "Issue description" section, but realized I forgot some now. I tested hopper input and Hopperhock input with the appropriate items for each block. All of these blocks accept hopper input, and no Hopperhock input: pulse mana spreader, mana spreader, elven mana spreader, gaia mana spreader, mana prisms, tiny potatoes, open crates, crafty crates, spark tinkerers, the red-stringed container, the red-stringed dispenser, and hovering hourglass.
